test/pkgs/the-Saurus/extra/thesaurus_ViewExtensionDefinition.xml

```diff
@@ -0,0 +1,4 @@
+﻿<ViewExtensionDefinition>
```

Comment:
this is minor but in the future should we setup a `test/extensions` folder for extensions even though they are technically still packages just for clarity?

Reply:
If you would like to update the tests 😉 .